pkg/filesystem/cloudfilesystem/overlay2.go

```diff
@@ -92,7 +92,8 @@ func mountRootfs(ipList []string, target string, cluster *v2.Cluster, initFlag b
 func unmountRootfs(ipList []string, cluster *v2.Cluster) error {
 	var (
 		clusterRootfsDir = common.DefaultTheClusterRootfsDir(cluster.Name)
-		execClean        = fmt.Sprintf("/bin/bash -c "+common.DefaultClusterClearBashFile, cluster.Name)
+		cleanFile        = fmt.Sprintf(common.DefaultClusterClearBashFile, cluster.Name)
+		execClean        = fmt.Sprintf("chmod +x %s && /bin/bash -c "+cleanFile, cleanFile)
```

Comment:
execClean = fmt.Sprintf("chmod +x %s && /bin/bash -c %s", cleanFile)

better ?

Reply:
> execClean = fmt.Sprintf("chmod +x %s && /bin/bash -c %s", cleanFile)
> 
> better ?

that's right, my mistake